community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -483,15 +599,22 @@ def main():
             sync_slurm()
         except Exception:
             log.exception("failed to sync instances")
+
         try:
             sync_placement_groups()
         except Exception:
             log.exception("failed to sync placement groups")
+
         try:
             update_topology(lkp)
         except Exception:
             log.exception("failed to update topology")
 
+        try:
+            sync_maintenance_reservation(lkp)
```

Comment:
Either let's not add it yet, or add a knob so user can turn it off/

Reply:
Commented out. Adding a knob sounds like traveling from blueprint.